data/faq.html

```diff
@@ -95,39 +95,48 @@ <h3 lang="es">FAQ</h3>
 <li><a href="#mapdata" class="link" lang="en">Map data</a>
     <a lang="ru" href="#mapdata">Картографические данные</a>
     <a lang="de" href="#mapdata">Kartendaten</a>
-    <a lang="es" href="#mapdata">Datos de los mapas</a></li>
+    <a lang="es" href="#mapdata">Datos de los mapas</a>
+    <a lang="fr" href="#mapdata">Données de la carte</a></li>
 <li><a href="#mapeditor" class="link" lang="en">Map editor</a>
     <a lang="ru" href="#mapeditor">Редактор карты</a>
     <a lang="de" href="#mapeditor">Karten-Editor</a>
-    <a lang="es" href="#mapeditor">Editor de mapas</a></li>
+    <a lang="es" href="#mapeditor">Editor de mapas</a>
+    <a lang="fr" href="#mapeditor">Éditeur de carte</a>
```

Comment:
But you use "Données de la carte" above. I can a sense in which this is different, just had to double-check.